staff/lbaella/JavaScript/calculation.js

```diff
@@ -0,0 +1,15 @@
+/*
+Calculation
+Write a program that writes in the console all the multiples of 23 less than 500 and at the end writes the sum of all of them
+*/
+
+function calculation() {
+    var sum = 0;
+    for (var i=0; i<=500; i+=23) {
```

Comment:
We need to return 2 things: the list of numbers and the sum of all of them
Give it another try!